tracer/src/Datadog.Trace/DuckTyping/DuckType.Methods.cs

```diff
@@ -1047,20 +1046,58 @@ internal static Type AddIlToDuckChain(LazyILGenerator il, Type genericType, Type
                     {
                         DuckTypeException.Throw($"CreateCache<{genericType}>.CreateFrom<{fromType}>() cannot be found!");
                     }
+
+                    il.Emit(OpCodes.Call, getProxyMethodInfo);
+                }
+                else if (genericType.IsGenericType && genericType.GetGenericTypeDefinition() == typeof(Nullable<>))
+                {
+                    // Support for Nullable<T>
+                    var argGenericType = genericType.GenericTypeArguments[0];
+                    var getProxyMethodInfo = typeof(CreateCache<>)
+                                        .MakeGenericType(argGenericType)
+                                        .GetMethod("Create");
```

Comment:
Should we use `CreateFrom` here seeing as it's a value type? 🤔

Reply:
![image](https://github.com/DataDog/dd-trace-dotnet/assets/69803/0d4e5517-65e4-4047-b3d1-b799c6e0024f)

Reply:
Oh right, I'm getting my `from` and `to`s confused 🤦‍♂️